inclusive-design/wecount.inclusivedesign.ca
# fix: improve visual appearance of the website (resolves #322)

## Description
Changed font sizes for headings and side menus to improve visual appearance of the website.


## Steps to test

1. Go to deploy preview
2. Navigate to the About page
3. Observe header nav
4. Observe side menu

**Expected behavior:** 
The font changes should improve visual appearance of the website.
## Additional information
N/A


## Related issues


closes #322
